lib/codegen/generator.dart

```diff
@@ -292,7 +292,7 @@ class _DeclarationCode extends ConstExpression {
 
   List<String> get librariesUsed => []
     ..addAll(type.librariesUsed)
-    ..addAll(annotations.expand((a) => a.librariesUsed));
+    ..addAll(annotations.expand((a) => a.librariesUsed) as Iterable<String>);
```

Comment:
I added the type annotation and got another error.
ERROR: The type '((ConstExpression) → Iterable<dynamic>) → Iterable<dynamic>' is declared with 0 type parameters, but 1 type arguments were given ([smoke] lib/codegen/generator.dart:295)

Is there some flag I need to set to enable generics? (Obviously I have strong mode enabled.)